README.md

```diff
@@ -30,13 +30,19 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the DOM?
 
+Document object model- is an interface for html that allows the document to be manipulated
+
 2. What is an event?
+Events represent user actions that are executed through functions.
 
 3. What is an event listener?
+An event listener is the function that will be executed when an 'event' takes place
 
 4. Why would we convert a NodeList into an Array?
+When creating certain functions/methods nodelist do not register as some programs expect arrays
```

Comment:
This is where we would need to think about the reasons why we would have an HTML Collection vs reasons why we would have a NodeList. NodeList is an array like object that -- prebaked -- gives us the forEach method. In order to use other array methods we have to convert it to an array.